src/display/api.js

```diff
@@ -3147,6 +3156,26 @@ class PDFObjects {
     }
     this.#objs = Object.create(null);
   }
+
+  [Symbol.iterator]() {
```

Comment:
Why not using a stared function and yield here ?

Reply:
I'm probably missing something really obvious, but what's the advantage of doing that?

Reply:
It should make the code slightly simpler (and likely slightly smaller):
 - no explicit creation of `entries` (maybe it's done internally, I don't know)
 - no need to explicitly use the iterator protocol.
I can understand it's a matter of taste, so feel free to ignore.

Reply:
You're right, that's a little bit shorter and we can also get rid of `Object.entries(...)` as well.
Thank you!